examples/TestReceiveVideo/TestReceiveVideo.sln

```diff
@@ -0,0 +1,25 @@
+﻿

```

Comment:
Is it preferable to have a separate solution, or use the main one at the root of the repository? I am genuinely asking, I don't know what's best. Why did you go with a separate solution in the first place?